### PR TITLE
Dev utpal

### PR DESCRIPTION
**Pull request by utpalmishra**
Fixed few files to sanitize Q/TESTS directory

### CODE REVIEW

File: TESTS/test_lamping.lua [hunk 1]
```diff
@@ -50,7 +50,8 @@ local tests = {}
 tests.t1 = function ()
   -- TODO Create some data for agct
   local datadir = os.getenv("Q_SRC_ROOT") .. "/TESTS/"
-  agct = Q.load_csv(datadir .."agct.csv")
+  local M = dofile(datadir .. "meta_data_lamping.lua")
+  agct = Q.load_csv(datadir .."agct.csv", M, { is_hdr = true, use_accelerator = false})
```
Comment (mardikark-gslab) [inline TESTS/test_lamping.lua]: This 'agct' can be local, not required to be global

File: TESTS/test_view_meta.lua [hunk 2]
```diff
@@ -1,6 +1,7 @@
 -- FUNCTIONAL
 local Q = require 'Q'
-require 'Q/UTILS/lua/strict'
+-- "TO VIEW META VECTORS ARE REQUIRED TO BE GLOBAL, HENCE STRICT CLAUSE CANNOT BE APPLIED"
+--require 'Q/UTILS/lua/strict'
```
Comment (mardikark-gslab) [inline TESTS/test_view_meta.lua]: Please uncomment the "require 'Q/UTILS/lua/strict" line.

File: TESTS/test_lamping.lua (hunk 1 above)
Comment (utpalmishra) [reply TESTS/test_lamping.lua]: Made the changes

File: TESTS/test_dotprod_parallel.lua [hunk 3]
```diff
@@ -48,10 +48,10 @@ tests.t1 = function ()
   collectgarbage()
 	end
 -- dbg()
-	X:eval()
-	ysubp:eval()
+	--X:eval()
+	--ysubp:eval()
 
-	local b = Q.sum(Q.vvmul(X, ysubp))
+	b = Q.sum(Q.vvmul(X, ysubp))
```
Comment (mardikark-gslab) [inline TESTS/test_dotprod_parallel.lua]: keep this variable 'b' as local

**Pull request merged**